### PR TITLE
Loops docs page

### PR DESCRIPTION
**Pull request by danrowden**

### CODE REVIEW

File: src/connections/destinations/catalog/loops/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+# The end name should be similar to `Slack  Destination`
+title: Loops
+hide-boilerplate: true
+hide-dossier: true
+---
+
+{% include content/plan-grid.md name="actions" %}
```
Comment (askkaz) [inline src/connections/destinations/catalog/loops/index.md:8]: @danrowden what does this line do

File: src/connections/destinations/catalog/loops/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+[Loops](https://loops.so){:target:="_blank"} is a modern email platform for SaaS, a better way to send marketing and transactional email.
+
+You can use this Segment integration to create and update your Loops contacts as well as trigger email sending via events.
+
+<!-- This include describes the requirement of A.js 2.0 or higher for Actions compatibility, and is required if your destination has a web component. -->
```
Comment (askkaz) [inline src/connections/destinations/catalog/loops/index.md]: i dont think we have a web component so we could leave this out

File: src/connections/destinations/catalog/loops/index.md (hunk 1 above)
Comment (danrowden) [reply src/connections/destinations/catalog/loops/index.md:8]: It adds this box into the page.

Screenshot 2023-09-29 at 08 21 29

Here's the template I worked from: https://github.com/segmentio/action-destinations/blob/main/docs/doc-template-new.md?plain=1

Comment (danrowden) [reply src/connections/destinations/catalog/loops/index.md:8]: I'm not actually sure. It's in [the template that Segment provide](https://github.com/segmentio/action-destinations/blob/main/docs/doc-template-new.md?plain=1) and in a lot of pages in the Destinations section of the docs, but doesn't seem to ever render on the web.

Here's an example: https://segment.com/docs/connections/destinations/catalog/actions-canny/

**Comment by danrowden** [comment]
I found another docs template from reading through the Segment docs (https://raw.githubusercontent.com/segmentio/segment-docs/develop/templates/partners/destination-new.md) and that also refrences the include you were asking about. I guess it's necessary to have there.

I've updated the PR to include an extra paragraph.

**Pull request merged**